### DISCUSSION TITLE
Formulae for Xcode templates

### DISCUSSION THREAD

**Discussion by jozefizso** [Homebrew/discussions#4564] [Writing Formulae/Casks]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

I would like to create a formulae for an Xcode template (`.xctemplate`) which would be installed into user folder at `~/Library/Developer/Xcode/Templates`.

Is it possible to extend formulae to manage data in folders other that `/usr/local/Cellar`?

What would be alternative approach? To install the template into Cellar and symlink it to Xcode Template folder?

**Comment by SMillerDev** [comment]
> Is it possible to extend formulae to manage data in folders other that /usr/local/Cellar?

Yes, on ARM it can be /opt/Homebrew

> What would be alternative approach? To install the template into Cellar and symlink it to Xcode Template folder?

Having the user do that, or making a package and installing it as a cask.

**Comment by jozefizso** [comment]
Thanks @SMillerDev, I will take a look at Casks.
